stories/Molecules/Sidebar-data/sidebardata.scss

```diff
@@ -3,13 +3,14 @@
 @import '../../assets/scss/variables';
 
 .accordion-navbar__item--headertext {
+  @extend %img_none;
+
   border: none;
-  font-family: $Font-Family-ProximaBold;
   font-size: $font-size-16;
+  // font-family: $Font-Family-ProximaNova;
```

Comment:
@harishkumar-srijan Remove this commented code

Reply:
Done